ProjectSettings/DynamicsManager.asset

```diff
@@ -4,12 +4,12 @@
 PhysicsManager:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Gravity: {x: 0, y: -9.81, z: 0}
```

Comment:
We have a script to change the settings in our demos.  I'll get you sorted offline.